### DISCUSSION TITLE
Add a time picker for the date type in the Form component

### DISCUSSION THREAD

**Discussion by theochr72** [sqlpage/SQLPage#717] [Q&A]
Hello,

Currently, the date type in the form component does not allow users to select a time.
Would it be possible to add a time picker to enable combined date and time input?

Thank you for sqlpage, it is amazing!

**Comment by lovasoa** [comment]
Hello and welcome to SQLPage, @theochr72 !

What are you building ?

SQLPage  does already support all browser input types, including `datetime-local`, which is probably what you were looking for:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/datetime-local
